pkgs/servers/sql/postgresql/ext/pg-semver.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  postgresql,
+  postgresqlTestExtension,
+  testers,
+  buildPostgresqlExtension,
+}:
+
+buildPostgresqlExtension (finalAttrs: {
+  pname = "pg-semver";
+  version = "0.40.0";
+
+  buildInputs = postgresql.buildInputs;
```

Comment:
Are those really needed?
I can compile the package without this. `buildPostgresqlExtension` already sets buildInputs.